test/e2e/actuals_test.go

```diff
@@ -1406,6 +1406,22 @@ func workNamespaceRemovedFromClusterActual(cluster *framework.Cluster) func() er
 	}
 }
 
+// configMapRemovedFromClusterActual returns a function that verifies the ConfigMap has been removed from the specified cluster.
+func configMapRemovedFromClusterActual(cluster *framework.Cluster) func() error {
```

Comment:
There's an existing function for it right below: `namespacedResourcesRemovedFromClusterActual`.